code/evaluation/score_models.r

```diff
@@ -6,6 +6,8 @@ library(tidyr)
 library(lubridate)
 library(here)
 library(readr)
+source(here("code", "config_utils", "get_forecast_targets.R"))
```

Comment:
would it make more sense to add this to (a fork of) `covidHubUtils` instead?

Reply:
My vote is to merge this first and then, I will submit a PR to covidHubUtils

Reply:
OK